### PR TITLE
325: DH tweaks

### LINKED ISSUE #325: DH field layout - full view
- [x] Review and confirm field display order (@michael-wynne-wsu)
- [x] Consider adding sub-divisions to the sidebar. Even just a more prominent horizontal divide. Eg: core mukurtu fields || additional metadata || rights.
- [x] Broader question: that sidebar is getting extremely full, are there any other display options to improve this?
- ~~If possible, only display node title in sidebar when there is a community record/multi-page item. Not necessary for a standalone DH item. LOW priority at this time.~~ Moved to a standalone issue.
- ~~See also PDF viewer #132~~

### PR DESCRIPTION
**Pull request by pjudge**
This PR closes #325 

- Rearranges the order of fields on DH, both main content and sidebar
- Splits the sidebar into 3 block-like sections
- Visually hides the map label

### CODE REVIEW

**Comment by michael-wynne-wsu** [comment]
Only one change requested - there can be an empty sidebar block if (I think) there are no fields that would go in that block. Looks great otherwise!

Screenshot 2024-10-28 at 8 24 02 AM

**Comment by michael-wynne-wsu** [comment]
@pjudge Just confirming that this is still open because of the fieldgroups issues.

I have no issues with @nick-deer merging it if you'd rather continue that on a separate branch.

**Comment by pjudge** [comment]
@nick-deer I had to add a patch for the Paragraphs module. I opened a PR for the Template repo: https://github.com/MukurtuCMS/Mukurtu-CMS-v4-Project-Template/pull/4

The gitpod is erroring, and I think it might be because the template PR should be merged first?

**Comment by pjudge** [comment]
@nick-deer Thanks for merging that -- I just realized that that wouldn't help, and indeed it did not. Are you familiar with the error?

**Comment by nick-deer** [comment]
@pjudge I merged the template changes, but you're right that the gitpod is still erroring. I see that you put the paragraphs patch file inside a new patches directory at web/profiles/contrib/mukurtu-cms/patches/paragraphs-118.patch, but I couldn't find it in the Mukurtu 4 code repo or the v4 template's patches directory. 

I'll try moving that patch to the template's patches directory and see if that works. I'll also add the patch to the gitpod composer yml file

**Comment by nick-deer** [comment]
Okay I fixed the gitpod errors so it's building again, but the empty paragraph on the sidebar is still showing up for me...

File: mukurtu-gitpod-site.composer.json [hunk 1]
```diff
@@ -151,14 +151,6 @@
                 "type:drupal-custom-theme"
             ]
         },
-        "patches": {
-            "drupal/message_ui": {
-                "Fix installation": "https://git.drupalcode.org/project/message_ui/-/merge_requests/2.diff"
-            },
-            "drupal/paragraphs": {
```
Comment (pjudge) [inline mukurtu-gitpod-site.composer.json]: Hi @nick-deer This looks like it removes the patch installation from gitpod. I think we probably need this in order for the patch to apply. I added the patch to the branch in an earlier commit 

https://github.com/MukurtuCMS/Mukurtu-CMS/blob/ba2a9a288f8b1ab0383e287049a10d86dcb062fb/patches/paragraphs-118.patch

Comment (nick-deer) [reply mukurtu-gitpod-site.composer.json]: Oh my gosh you're right I'm sorry I missed that! Let me revert my changes on the template repo and I'll merge this PR and PR 604

**Pull request merged**